### ISSUE TITLE
[Feature Request] Electron-compatible and -specific features

### ISSUE THREAD

**Issue by hanayashiki** [morethanwords/tweb#12]
**Is your feature request related to a problem? Please describe.**
Hi, I find this client of telegram really beautiful and smooth. It would be a waste if it only works in browser! I am wraping an electron version of it under: https://github.com/hanayashiki/tweb-electron It now works (mostly) nicely on my MacBook Pro. However, I find only implementing a simple wrapper is not enough.

**Describe the solution you'd like**
Electron is not just a browser without the URL bar. 

There are numbers of differences that need to keep in mind. For example, when click on a address, it should pop up the user's browser instead of opening an electron window.

Plus, electron gives us the power to electron itself,  lower network layers and OS apis, so features like MTProto proxy, Socks5 proxy, local file system access, etc, can be implemented too.

**Describe alternatives you've considered**

No.

**Additional context**

No.

**Comment by morethanwords** [comment]
You're totally right, there should be an Electron version of it. I have it in my to-do list.
